projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.scss

```diff
@@ -0,0 +1,16 @@
+:host {
+  display: flex;
+  height: 100%;
+  overflow: hidden;
+}
+
+.ngx-row-accordion {
+  display: flex;
+  width: 500px;
+  overflow: hidden;
+}
+
+.title {
+  position: relative;
+  z-index: 20;
```

Comment:
Is that needed?

Reply:
Hey Maxime, yes this is needed to ensure the panel remains on top and visible throughout the animation. I've added a comment in the code to explain and reduced the z-index value down. 👍